vaadin-notification-flow-demo/src/main/java/com/vaadin/flow/component/notification/demo/NotificationView.java

```diff
@@ -141,4 +143,41 @@ private void addStyledNotificationContent() {
         button.setId("styled-content-notification-button");
         addCard("Notification with styled content", button);
     }
+
+    private void createThemeVariants() {
+
+        //@formatter:off
+        // begin-source-example
+        // source-example-heading: Theme Variants
+        Notification primary = new Notification("Primary theme variant", 3000);
+        primary.addThemeVariants(NotificationVariant.LUMO_PRIMARY);
```

Comment:
Probably buttons should be added to the demos as it's done for the webcomponent:
https://cdn.vaadin.com/vaadin-notification/1.3.0/demo/#notification-theme-variants-demos

Reply:
Demos are now aligned with the web component, using buttons with theme variants inside the notifications.

However, for the buttons to be styled correctly, we need to wait for new releases of `vaadin-button` and `vaadin-notification` containing fixes for styling buttons which are not direct children of the notification: https://github.com/vaadin/vaadin-button/pull/129
This is needed because in Flow, the notification wraps the contents inside a `flow-component-renderer` and a `div`.

Reply:
Done.